### PR TITLE
Alpha simon bugfixes

### PR DESCRIPTION
**Pull request by simonlsk**
Mount works with oauth and cli.
Improved logs

### CODE REVIEW

File: dagshub/streaming/mount.py [hunk 1]
```diff
@@ -2,6 +2,9 @@
 import logging
 import os
 import platform
+
+logger = logging.getLogger(__name__)
+
```
Comment (kbolashev) [inline dagshub/streaming/mount.py:7]: Redundant, also called later

File: dagshub/streaming/mount.py [hunk 2]
```diff
@@ -28,33 +33,46 @@ def __init__(self,
...
+        logger.debug(f"open - path: {path}, flags: {flags}")
         if path == Path(self.fs.project_root / SPECIAL_FILE):
             return SPECIAL_FILE_FH
-        self.fs.open(path).close()
-        return os.open(path, flags, dir_fd=self.fs.project_root_fd)
+        try:
+            self.fs.open(path).close()
+        except FileNotFoundError:
+            raise FuseOSError(errno.ENOENT)
+        logger.debug("finised fs.open")
```
Comment (kbolashev) [inline dagshub/streaming/mount.py:62]:
```suggestion
        logger.debug("finished fs.open")
```

File: dagshub/streaming/mount.py [hunk 3]
```diff
@@ -91,13 +113,15 @@ def mount(debug=False,
...
+          password: Optional[str] = None,
+          token: Optional[str] = None):
     logging.basicConfig(level=logging.DEBUG)
     fuse = DagsHubFUSE(project_root=project_root, repo_url=repo_url, branch=branch, username=username,
-                       password=password)
-    print(
+                       password=password, token=token)
+    logger.debug(
         f'Mounting DagsHubFUSE filesystem at {fuse.fs.project_root}\n'
         f'Run `cd .` in any existing terminals to utilize mounted FS.')
```
Comment (kbolashev) [inline dagshub/streaming/mount.py:123]: This should be either a print or a warning. User must see this

**Review by kbolashev — APPROVED** [review]
Linter errors + "Mounting DagsHubFUSE..." part might have accidentally been changed to a logger.debug.
Other than that lgtm

File: dagshub/streaming/mount.py (hunk 3 above)
Comment (simonlsk) [reply dagshub/streaming/mount.py:123]: Sorry, aggressive renaming.

**Pull request merged**